plugins/destination/postgresql/client/migrate.go

```diff
@@ -399,3 +402,25 @@ func (c *Client) removeUniqueConstraint(ctx context.Context, table *schema.Table
 func getPKName(table *schema.Table) string {
 	return table.Name + "_cqpk"
 }
+
+func (c *Client) createPerformanceIndexes(ctx context.Context, table *schema.Table) error {
+
+	columns := []string{"_cq_source_name", "_cq_sync_time"}
+
+	for _, col := range columns {
+		if table.Columns.Get(col) == nil {
+			c.logger.Debug().Msgf("no performance indexes created because %s column is not present", col)
```

Comment:
Below `columns[1]` is not used at all, why are we checking for its existence? Are we trying to determine if the `_cq_source_name` col is legit/cq-owned?

Reply:
columns[1] should be used.

fixed in [d58dffc](https://github.com/cloudquery/cloudquery/pull/18741/commits/d58dffc440389daa442d8c9a29757cdc0ce609b5)